src/omnicore/rules.cpp

```diff
@@ -318,8 +319,9 @@ bool ActivateFeature(uint16_t featureId, int activationBlock, int transactionBlo
     const CConsensusParams& params = ConsensusParams();
 
     // check activation block is allowed
-    if ((activationBlock < (transactionBlock + params.MIN_ACTIVATION_BLOCKS)) ||
-        (activationBlock > (transactionBlock + params.MAX_ACTIVATION_BLOCKS))) {
+    if ( ((transactionBlock + params.MIN_ACTIVATION_BLOCKS) > activationBlock) ||
```

Comment:
Wha, very, very, good catch! I used `[0, std::numeric_limits<int>::max()]` to express "unlimited".